themes/reactos/layouts/shortcodes/figure.html

```diff
@@ -5,7 +5,7 @@
 -->
 <!-- use either src or link-thumb for thumbnail image -->
 {{- $thumb := .Get "src" | default (printf "%s." (.Get "thumb") | replace (.Get "link") ".") }}
-<figure {{ with .Get "class" }}class="{{.}}"{{ end }} itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+<figure {{ with .Get "class" }}class="{{.}}"{{ end }} itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject" {{ with .Get "width" }}style="max-width:{{.}}"{{end}}>
```

Comment:
What happens if width is not specified, empty tag or completely omitted?

Reply:
Everything between `{{ with ...` and `{{end}}` is simply left out. See for example the source of https://pr61.web-content.reactos.org/project-news/reactos-0413-released/

It should be noted that this was previously part of our figure shortcode and imported from upstream source (where it's also an optional attribute).
@Extravert-ir removed it when fixing image responsiveness on our website.
I'm adding it back, because I need a way to specify how much space an image floating next to text may consume.